src/ros_video_components/src/ros_voltage_meter.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Date Started: 8/09/18
+ * Original Author: Sajid Ibne Anower
+ * Editors:
+ * Purpose: Widget to display voltage of the batteries
+ * This code is released under the MIT License. Copyright BLUEsat UNSW 2018
+ * Trivia: This code was written on the plane from Sydney to Warsaw,
+ * on the way to participating in ERC2018
+ */
+#include "ros_video_components/ros_voltage_meter.hpp"
+
+void ROS_Voltage_Meter::setup(ros::NodeHandle * nh) {
+    volt_sub = nh->subscribe(
+        "/rover/volt",
```

Comment:
would something like `/sensors/voltmeter` be more portable? `/rover` is very generic as pretty much all topics could fall under it :P

Reply:
Yes, that's a good point. I have made that change.